src/bundle/Resources/public/scss/_buttons.scss

```diff
@@ -137,3 +137,24 @@
 .ez-btn--cotf-create {
     display: flex;
 }
+
+.ez-search-view {
+    .input-group-btn {
+        .ez-btn--search,
+        .ez-btn--filter {
+            .ez-icon {
+                margin-right: .25rem;
+                vertical-align: sub;
```

Comment:
Do you want to display an icon centered in the button?

Reply:
Not sure to follow, but yes.